spec/rails_helper.rb

```diff
@@ -12,6 +12,8 @@
 # use for there expectations, etc.
 ENV['IIIF_HOST_URL'] = 'localhost'
 
+ENV['USE_JWT_AUTH'] = 'Y'
```

Comment:
We should write tests to make sure a 403 is not returned when USE_JWT_AUTH is set to 'N' (or other value that is not 'Y')